src/import/customer-import.service.ts

```diff
@@ -82,6 +84,30 @@ export class CustomerImportService {
       });
     }
 
+    for await (const ordersBatch of this.orderImportService.getAllOrders(
+      period,
```

Comment:
when I saw `period` param I thought it's gonna be some object containing two dates (from / to), but it's just a number. Then a question pops up: that it is? Is it number of seconds? Minutes? Days? Timestamp in millis since 1.1.1970?

When I check the code it turns out it's number of days in the past / since. I would suggest to rename it to `daysInPast`, `numberOfDays` or something similar.
Or maybe even better: to accept the exact (iso) timestamp you'd like to fetch data from.